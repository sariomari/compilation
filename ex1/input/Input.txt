class void IntList
{
    int head;
    IntList tail;
}