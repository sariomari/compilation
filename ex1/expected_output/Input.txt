CLASS[1,1]
TYPE_VOID[1,7]
ID(IntList)[1,12]
LBRACE[2,1]
TYPE_INT[3,5]
ID(head)[3,9]
SEMICOLON[3,13]
ID(IntList)[4,5]
ID(tail)[4,13]
SEMICOLON[4,17]
RBRACE[5,1]